docs/xks/operator-guide/aws-azdo.md

```diff
@@ -0,0 +1,35 @@
+---
+id: aws-azdo
+title: AWS azure-devops
+---
+
+We use Azure DevOps to manage git repositories even if you are using AWS to deploy XKF and we run all our pipelines in Azure using VMSS just like in a Azure based XKF.
+
+The main difference is that we use the AWS service connection.
+First you need to install the addon to Azure DevOps using the Azure marketplace,
+you can find more information how [here](https://docs.aws.amazon.com/vsts/latest/userguide/getting-started.html).
+
+We will use two users when talking to AWS, one for our terraform pipelines and one for
+our gitops flow where we use Elastic Container Registry (ECR) to store the container images.
+
+## Terraform
+
+All this is possible to solve using Terraform but we currently don't have any terraform module to handle it so for now you can read about the manual steps.
+
+- Create a IAM user
+  - Tip put both these users under a specific path, for example CI
+  - Save the credentials in a safe place
+- Attach IAM policy
+  - AmazonEKSClusterPolicy
+  - AdministratorAccess
```

Comment:
Let's not leave anything to chance:
```bash
aws iam create-user --user-name xenit-terraform-user --path /ci/
aws iam attach-user-policy --user-name xenit-terraform-user --policy-arn arn:aws:iam::aws:policy/AdministratorAccess
aws iam create-access-key --user-name xenit-terraform-user
```
Also, since we have administrator access, nothing more should be needed?

Reply:
Since it's the CI pipeline that creates other resources the CI user needs to have admin access.
I don't want to maintain any commands here, in the long run we will create a Terraform module for it.
From my point of view this is a hint on what needs to get done, not perfect but at least something.